server/data_structures/shared_datafacade.hpp

```diff
@@ -418,13 +419,15 @@ template <class EdgeDataT> class SharedDataFacade final : public BaseDataFacade<
         }
 
         return m_static_rtree->second->IncrementalFindPhantomNodeForCoordinate(
-            input_coordinate, resulting_phantom_node_vector, number_of_results);
+            input_coordinate, resulting_phantom_node_vector, number_of_results, 1100, bearing, range);
```

Comment:
Same here with hard-coding the max. distance. I understand how this is required here for being able to pass bearing and range with values other than the default ones. But maybe reorder the arguments then?